netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -745,10 +765,94 @@ _net_ebpf_sock_addr_clean_up_security_descriptor()
     }
 }
 
-static void
-_net_ebpf_sock_addr_initialize_globals()
+void
+_net_ebpf_ext_uninitialize_blocked_connection_contexts()
 {
-    InitializeListHead(&_net_ebpf_ext_connect_context_list);
+    KIRQL old_irql = ExAcquireSpinLockExclusive(&_net_ebpf_ext_sock_addr_blocked_contexts.lock);
+
+    // Cleanup all in use connect contexts.
```

Comment:
```suggestion
    // Clean up all in use connect contexts.
```
nit: "cleanup" is a noun.  "clean up" is a verb.